apps/ex_wire/lib/ex_wire/sync.ex

```diff
@@ -17,13 +17,18 @@ defmodule ExWire.Sync do
 
   alias Block.Header
   alias Blockchain.Block
+  alias Blockchain.Blocktree
+  alias Blockchain.Blocktree.State
+  alias Blockchain.Chain
   alias ExWire.Config
   alias ExWire.Packet.{BlockBodies, BlockHeaders, GetBlockBodies, GetBlockHeaders}
   alias ExWire.PeerSupervisor
-  alias ExWire.Struct.{BlockQueue, Peer}
-  alias Blockchain.{Blocktree, Chain}
-  alias Blockchain.Blocktree.State
-  alias MerklePatriciaTree.{CachingTrie, DB.RocksDB, Trie, TrieStorage}
+  alias ExWire.Struct.BlockQueue
```

Comment:
I see this has moved from the `{}` alias syntax to an alias per line. Is there a reason this is desirable or is this mostly a stylistic choice?